addons.html

```diff
@@ -92,7 +94,8 @@ <h3>Apps and Utilities:</h3>
                         href="https://github.com/tympanix/Electorrent">Electorrent</a>,&nbsp; <a
                         href="https://github.com/tremc/tremc">tremc</a>,&nbsp; <a
                         href="https://github.com/rndusr/stig">stig</a>,&nbsp; <a
-                        href="https://github.com/dylanaraps/torque">torque</a>
+                        href="https://github.com/dylanaraps/torque">torque</a>,&nbsp; <a
+                        href="https://github.com/openscopeproject/TrguiNG">TrguiNG</a>
```

Comment:
If it's a web UI replacement (and so doesn't work w/o a web browser), I guess it's not standalone? I'd leave the entry above and drop the one here.

Reply:
It's both standalone and web ui replacement.

Reply:
Uses native web view controls (and webkit on GNU/Linux), I see it now. I guess that's how standalone looks like these days, okay then... ;)